src/flamenco/vm/fd_vm_interp_core.c

```diff
@@ -818,24 +827,17 @@
     FD_VM_INTERP_STACK_PUSH;
 
     ulong vaddr = reg[ imm & 15U ];
-    ulong region = vaddr >> 32;
-    ulong align  = vaddr & 7UL;
-    ulong target_pc = ((vaddr & FD_VM_OFFSET_MASK)/8UL) - text_word_off;
 
-    /* Note: BRANCH_END will implicitly handle a pc that fell outside
-       the text section (below via unsigned wraparoud or above) as
-       sigtext.
+    /* Notes: Agave checks region and target_pc before updating the pc.
+       To match their state, we do the same, even though we could simply
+       update the pc and let BRANCH_END fail.
+       Also, Agave doesn't check alignment. */
```

Comment:
Is this still true? I think this was fixed [here](https://github.com/solana-labs/rbpf/pull/583/files). I can check with Alexander

Reply:
I guess it's useless given the truncated divide we perform below.